contrib/scripts/qt5.py

```diff
@@ -0,0 +1,66 @@
+import os
+import re
+import functools
+
+import Qt
+
+def convert( fileName ) :
+
+	with open( fileName ) as f :
+		text = "".join( f.readlines() )
+		#print text
```

Comment:
Thanks Ivan - I've addressed this in 26b572c, using argparse to add proper command line help.